### PR TITLE
Update dead link in extending.rst

### PR DESCRIPTION
**Pull request by danielmichaels**
> brief description what this PR will do, e.g. fixes broken dissection of XXX

### Improve Documentation

Reading over the docs, the link to [Wifitap](http://sid.rstack.org/static/articles/w/i/f/Wifitap_EN_9613.html) is dead. I found Wifitap's GitHub [repo](https://github.com/gdssecurity/wifitap/) and replaced the link with it.

### CODE REVIEW

**Comment by gpotter2** [comment]
The repo you've provided looks pretty dead :/
It seems that https://github.com/GDSSecurity/wifitap/pull/1 actually makes it work with recent Scapy versions, but I doubt it really does with master :/

I'll ask there if the project is dead. If it is, it might be a better idea to remove the link

**Comment by p-l-** [comment]
We are keeping that link for historical & sentimental reasons. I think we should just add a note to state that Wifitap is no longer updated... but I'd like to keep the current link.

**Comment by danielmichaels** [comment]
> We are keeping that link for historical & sentimental reasons. I think we should just add a note to state that Wifitap is no longer updated... but I'd like to keep the current link.

Does a change like this suit?

File: doc/scapy/extending.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ Here is another tool that will constantly monitor all interfaces on a machine an
     
     sniff(prn=arp_monitor_callback, filter="arp", store=0)
 
-For a real life example, you can check `Wifitap <http://sid.rstack.org/static/articles/w/i/f/Wifitap_EN_9613.html>`_.
+For a real life example, you can check `Wifitap <https://github.com/gdssecurity/wifitap/>`_. Sadly, Wifitap is no longer maintained but nonetheless demonstrates Scapy's Wi-Fi capabilities.
```
Comment (gpotter2) [inline doc/scapy/extending.rst]:
```suggestion
For a real life example, you can check `Wifitap <https://github.com/gdssecurity/wifitap/>`_. `[original archive] <https://web.archive.org/web/20180309055843/http://sid.rstack.org/static/articles/w/i/f/Wifitap_EN_9613.html>`_ Sadly, Wifitap is no longer maintained but nonetheless demonstrates Scapy's Wi-Fi capabilities.
```

Comment (danielmichaels) [reply doc/scapy/extending.rst]: :+1:

**Pull request merged**